tests/core/kernel-base-gadget-single-reboot/task.yaml

```diff
@@ -0,0 +1,133 @@
+summary: Verify that updating all of base, gadget and kernel only needs one reboot
```

Comment:
But there are actually a few reboots expected in the test? Is this maybe a copy of another test?

Reply:
Yep but that is intentional. If you look you'll see that the last couple of reboots are due to me reverting each snap we updated one-by-one, and we need to do this otherwise our spread checks will complain that system state has been modified.